src/debug.c

```diff
@@ -477,7 +477,7 @@ NULL
         };
         addReplyHelp(c, help);
     } else if (!strcasecmp(c->argv[1]->ptr,"segfault")) {
-        *((char*)-1) = 'x';
```

Comment:
Random write is an UB.

Reply:
Sanitizer will complain with attempts to write out of process' address space. So, 0, NULL or -1 fails as well. We can surpress with this attribute @zuiderkwast mentioned.